CommunicationLayer/CommunicationContainer.cpp

```diff
@@ -2,12 +2,9 @@
 #include <QUdpSocket>
 
 #include "../DataLayer/DataContainer.h"
-#include "CommunicationContainer.h"
-#include "CommDeviceControl/ConnectionController.h"
-#include "CommDeviceControl/CommDeviceManager.h"
```

Comment:
Good explanation, I'll accept that. No need for structural changes in the RadioCommDevice then.